tests/version/test_markers.py

```diff
@@ -2260,16 +2271,16 @@ def test_intersection_no_endless_recursion() -> None:
         ' and extra != "postgis"'
     )
     m2 = parse_marker(
-        'python_version > "3.12" and python_version < "3.13" or extra != "databricks"'
+        'python_version >= "3.12" and python_version < "3.13" or extra != "databricks"'
```

Comment:
This marker is actually empty and does not trigger a potential endless recursion anymore. I checked that the changed marker triggers the recursion error without the fix from #832.